### PR TITLE
update colours to reflect updated design and preliminary colour varia…

### PR DESCRIPTION
**Pull request by Furdook**
…bles

### CODE REVIEW

File: client/tailwind.config.js [hunk 1]
```diff
@@ -13,14 +12,13 @@ export default {
...
-        600: '#E0E0E0'
+      white: {
+        100: 'ffffff',
+        200: 'fefefe'
       },
-      error: {
-        500: '#E76C6C',
-        600: '#C53030'
-      }
+      error: {},
```
Comment (adelefelicia) [inline client/tailwind.config.js:19]: Curious: Why aren't colors set for these anymore?

**Review by adelefelicia — APPROVED** [review]

File: client/tailwind.config.js (hunk 1 above)
Comment (Furdook) [reply client/tailwind.config.js:19]: There was some uncertainty in which shades we will be using, so I've left them blank for now. :)

**Comment by EliHaugu** [comment]
Ah shit, looked at this one the day you posted it but forgot to post... Was gonna ask the same as Adele, why the error colors were removed but I see you answered above.
Btw are all the colors we're gonna use in 'accent'? I don't know tailwind too well but is that best practice? Is it easy to know which color to use where for you guys? If so I'm good with it, just curious as I haven't used this

**Comment by Furdook** [comment]
> Ah shit, looked at this one the day you posted it but forgot to post... Was gonna ask the same as Adele, why the error colors were removed but I see you answered above. Btw are all the colors we're gonna use in 'accent'? I don't know tailwind too well but is that best practice? Is it easy to know which color to use where for you guys? If so I'm good with it, just curious as I haven't used this

The thought is that our main colour palate is white, i.e. white-100 (#fff) and white-200 (#fefefe), and the accent colour in general is the different shades of green; as demonstrated on Figma. This makes it easy to find the shades of green you want to use by the shorthands bg-accent-800, text-accent-900, bg-accent-700, and so on. Following standards of a lighter shade with descending numbers, staying with the same colour group.

**Pull request merged**